index.js

```diff
@@ -21,6 +29,18 @@ function clean (db, done) {
       })
     },
     function (db, collections, cb) {
+      if (exclude === null) {
+        cb(null, db, collections)
+      } else {
+        // do not drop excluded collections
+        collections = collections.filter(function (coll) {
+          return (exclude.indexOf(coll.collectionName) === -1)
+        })
+
+        cb(null, db, collections)
+      }
+    },
```

Comment:
I would add it after this one: https://github.com/mcollina/mongo-clean/pull/1/files?diff=unified#diff-168726dbe96b3ce427e7fedce31bb0bcR26